sdk/keyvault/azure-security-keyvault/azure/security/keyvault/secrets/README.md

```diff
@@ -0,0 +1,231 @@
+# Azure Key Vault Secret client library for Python
+Azure Key Vault is a cloud service that provides a secure storage of secrets, such as passwords and database connection strings. Secret client library allows you to securely store and tightly control the access to tokens, passwords, API keys, and other secrets. This library offers operations to create, retrieve, update, delete, purge,backup, restore and and list the secrets and its versions.
+
+Use the secret client library to create and manage secrets.
+
+[Source code](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-security-keyvault/azure/security/keyvault/secrets) | [Package (Pypi)](TODO) | [API reference documentation](TODO) | [Product documentation](https://docs.microsoft.com/en-us/azure/key-vault/) | [Samples](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-security-keyvault/azure/security/keyvault/secrets/samples)
+## Getting started
+### Install the package
+Install the Azure Key Vault client library for Python with [pip](https://pypi.org/project/pip/):
+
+```Bash
+pip install azure-security-keyvault
+```
+
+### Prerequisites
+* An [Azure subscription](https://azure.microsoft.com/free/).
+* Python 2.7, 3.4 or later to use this package.
+* An existing Key Vault. If you need to create a Key Vault, you can use the [Azure Cloud Shell](https://shell.azure.com/bash) to create one with this Azure CLI command. Replace `<your-resource-group-name>` and `<your-key-vault-name>` with your own, unique names:
+
+```Bash
+az keyvault create --resource-group <your-resource-group-name> --name <your-key-vault-name>
+```
+
+### Authenticate the client
+In order to interact with the Key Vault service, you'll need to create an instance of the [KeyVaultClient](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-security-keyvault/azure/security/keyvault) class. You need a **vault url**, **client secret credentials (client id, client secret, tenant id)** and a [resource url](https://vault.azure.net) to instantiate a client object.
+
+ #### Create/Get credentials
+Use the [Azure Cloud Shell](https://shell.azure.com/bash) snippet below to create/get client secret credentials.
+
+ * Create a service principal and configure its access to Azure resources:
+    ```Bash
+    az ad sp create-for-rbac -n <your-application-name> --skip-assignment
+    ```
+    Output:
+    ```json
+    {
+        "appId": "fa7c8686-c4f5-4639-ab88-497ee122bad3",
+        "displayName": "myspn",
+        "name": "http://myspn",
+        "password": "30e5c628-a322-3497-8429-52f2f98788c7",
```

Comment:
Replace this value with an obvious placeholder like `random-password`. Also, if this was an actual password, the service principal is compromised; please destroy the secret or the service principal.